Flow.Launcher/ViewModel/MainViewModel.cs

```diff
@@ -285,9 +285,16 @@ public string QueryText
         /// but we don't want to move cursor to end when query is updated from TextBox
         /// </summary>
         /// <param name="queryText"></param>
-        public void ChangeQueryText(string queryText)
+        public void ChangeQueryText(string queryText, bool reQuery = false)
         {
-            QueryText = queryText;
+            if (QueryText!=queryText) 
+            {
+                QueryText = queryText;
+            }
+            else if (reQuery)
```

Comment:
I feel like this shouldnt be an 'else if', they should be two seperate statements

Reply:
Well actually, when QueryText is not equal to queryText, the requery will be triggered. Only when they are the same, we need manually requery.

Reply:
ohhh it's requeried in the setter method...

Reply:
lets add a comment in to make it clear

Reply:
i will commit

Reply:
I remember it is the optimization from fody xd